mantidproject/mantid
# Format release notes for 5.1.0 Diffraction, Muons, LSS and Spec

**Description of work.**

This PR covers the refactoing fo release notes for Diffraction, Muons, LSS and Spec. the refactoring of workbench, plot and framework will be covered in a different PR

**To test:**

Build docs and read the release notes to make sure they read in a way that makes sence and is in a consistent language.

Fixes #29507 

*This does not require release notes* because It is a change to the release note themselves.


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
